lib/rules/comment-empty-line-before/index.js

```diff
@@ -116,6 +117,16 @@ const rule = function (expectation, options) {
         return
       }
 
+      // Fix
+      if (context.fix) {
+        if (expectEmptyLineBefore) {
+          addEmptyLineBefore(comment, context.newline)
+        } else {
+          comment.raws.before = comment.raws.before.replace(/(\r?\n\s*\r?\n)+/g, context.newline)
```

Comment:
Could you, please, create a new helper function `removeEmptyLineBefore`? I didn't think about it at first, sorry.